src/runtime/runtime.ts

```diff
@@ -48,19 +48,19 @@ function matchMediaPreference() {
 // prettier-ignore
 function isKeyDownDarkMode(e: KeyboardEvent) {
 	const ok = (
-		e.ctrlKey &&
+		!e.ctrlKey &&
 		!e.altKey &&
-		(e.key.toLowerCase() === "d" || e.keyCode === 68)
+		(e.key.toLowerCase() === "`" || e.keyCode === 68)
```

Comment:
Hey this is pretty close but we need to change `68` to `192`. You can use keycode.info as a reference.